agent/runtime-attach/src/test/resources/applicationinsights.json

```
@@ -0,0 +1,7 @@
+{
+  "connectionString": "InstrumentationKey=00000000-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://dummy",
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  }
```

the others aren't needed for this
```suggestion
  "connectionString": "InstrumentationKey=00000000-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://dummy"
```